xml/ha_concepts.xml

```diff
@@ -732,38 +709,43 @@
     You can use either tool on any node in the cluster and they will be
     relayed to the DC. The DC will then replicate the CIB change to all
     cluster nodes.
    </para>
    <para>
-    Based on the information in the CIB, the PE then computes the ideal
-    state of the cluster and how it should be achieved. The PE feeds a list of
-    instructions to the DC. The DC sends commands via the
-    messaging/infrastructure layer which are received by the crmd peers on
-    other nodes. Each crmd uses its LRM (implemented as lrmd) to perform
-    resource modifications. The lrmd is not cluster-aware and interacts
-    directly with resource agents (scripts).
+    Based on the information in the CIB, the <systemitem
+    class="daemon">pacemaker-schedulerd</systemitem> then computes the ideal
+    state of the cluster and how it should be achieved. It feeds a list of
+    instructions to the DC. The DC sends commands via the messaging/infrastructure
+    layer which are received by the <systemitem
+    class="daemon">pacemaker-controld</systemitem> peers on
+    other nodes. Each of them uses its local resource agent executor (implemented
+    as <systemitem class="daemon">pacemaker-execd</systemitem>) to perform
+    resource modifications. The <systemitem
+     class="daemon">pacemaker-execd</systemitem>is not cluster-aware and interacts
+    directly with resource agents.
    </para>
    <para>
     All peer nodes report the results of their operations back to the DC.
     After the DC concludes that all necessary operations are successfully
     performed in the cluster, the cluster will go back to the idle state and
     wait for further events. If any operation was not carried out as
-    planned, the PE is invoked again with the new information recorded in
+    planned, the <systemitem class="daemon">pacemaker-schedulerd</systemitem>
+    is invoked again with the new information recorded in
     the CIB.
    </para>
    <para>
     In some cases, it may be necessary to power off nodes to protect shared
-    data or complete resource recovery. For this Pacemaker comes with a
-    fencing subsystem, stonithd. &stonith; is an acronym for <quote>Shoot
-    The Other Node In The Head</quote> and is usually implemented with a
-    remote power switch. In Pacemaker, &stonith; devices are modeled as
-    resources (and configured in the CIB) to enable them to be easily
-    monitored for failure. When clients detect a failure they send a
-    request to stonithd, which then fences the failed node.
+    data or complete resource recovery. In a Pacemaker cluster, the implementation
+    of node level fencing is &stonith;. For this, Pacemaker comes with a
+    fencing subsystem, <systemitem class="daemon">pacemaker-fenced</systemitem>.
+    &stonith; devices have to be configured as cluster resources (fencing
+    agents) in the CIB, because this allows to monitor them. When clients detect a
```

Comment:
"because this allows to monitor them" -> "This also allows to monitor the fencing devices. "

Reply:
Changed.